### ISSUE TITLE
Toggle "Featured" tab sometimes works only halfway

### ISSUE THREAD

**Issue by ckrey** [owntracks/ios#382]
via @jpmens 

![zz2746e721](https://cloud.githubusercontent.com/assets/60706/13048653/97c765fc-d3e7-11e5-93ea-18d6e8e267b0.jpg)

**Comment by jpmens** [comment]
screenshot edited

**Comment by ckrey** [comment]
I think I found the problem

**Comment by jpmens** [comment]
Without having published the `action` cmd, I just opened 8.6.5 and see space for the Featured tab, but no icon on it (as in above screenshot). Area is not clickable either.

What just occurs to me as I type this: is this possibly due to my *retained* message (you recall, because of having to test at a coffee shop?) :smile: 

```
owntracks/jpm/5s/cmd {"action": "action", "url": "https://www.google.de", "_type": "cmd", "tst": 1455348438}
```
